src/amp-story-player/amp-story-player-impl.js

```diff
@@ -546,11 +555,12 @@ export class AmpStoryPlayer {
             );
           });
 
-          const customControls =
-            this.playerConfig_ && this.playerConfig_['controls'];
-
-          if (customControls) {
-            messaging.sendRequest('customDocumentUI', customControls, false);
+          if (this.playerConfig_) {
```

Comment:
Nit: maybe `&& this.playerConfig_.controls` so we don't send an empty message if later a publisher specifies `behavior` only

Reply:
Yep good catch 👍